### PR TITLE
WIP: polish contract code

### PR DESCRIPTION
**Pull request by anna-carroll**
- Add comments for each section of contract code
- Add Natspec documentation for each function 
- Ensure events have proper variables included
- Edit variable names for clarity

### CODE REVIEW

File: solidity/optics-core/libs/Message.sol [hunk 1]
```diff
@@ -21,30 +21,30 @@ library Message {
...
+     * @param _destinationDomain Domain of destination chain
+     * @param _recipientAddress Address of recipient on destination chain as bytes32
+     * @param _messageBody Raw bytes of message body
      * @return Formatted message
      **/
     function formatMessage(
-        uint32 _origin,
-        bytes32 _sender,
+        uint32 _originDomain,
+        bytes32 _senderAddress,
```
Comment (prestwich) [inline solidity/optics-core/libs/Message.sol:34]: I generally don't like renaming these as `____Address` as it's _not_ an address in EVM parlance. it makes this feel misleading.

File: solidity/optics-core/contracts/Common.sol [hunk 2]
```diff
@@ -11,33 +11,51 @@ import {ECDSA} from "@openzeppelin/contracts/cryptography/ECDSA.sol";
...
+
+    // Address of bonded Updater
     address public updater;
-    /// @notice Current state of contract
+    // Current state of contract
     States public state;
-    /// @notice Current root
-    bytes32 public current;
+    // The latest root that has been signed by the Updater
+    bytes32 public current; // TODO: "latestUpdatedRoot"
```
Comment (prestwich) [inline solidity/optics-core/contracts/Common.sol:42]: latestCommittedRoot

**Review by prestwich — CHANGES_REQUESTED** [review]

File: solidity/optics-core/contracts/Common.sol (hunk 2 above)
Comment (anna-carroll) [reply solidity/optics-core/contracts/Common.sol]: https://github.com/celo-org/optics-monorepo/pull/680/commits/9190c245b98467fb75401028bd8ef47e613ac8ec

File: solidity/optics-core/libs/Message.sol (hunk 1 above)
Comment (anna-carroll) [reply solidity/optics-core/libs/Message.sol]: https://github.com/celo-org/optics-monorepo/pull/680/commits/9190c245b98467fb75401028bd8ef47e613ac8ec

**Pull request merged**